product_docs/docs/epas/15/reference/sql_reference/02_the_sql_language/02_data_types/array_types.mdx

```diff
@@ -4,6 +4,12 @@ title: "Array types"
 
 PostgreSQL allows the columns of a table to be defined as variable-length multidimensional arrays. Arrays of any built-in or user-defined base type, enum type, or composite type can be created. Arrays of domains aren't yet supported. 
 
+The following table lists the available array types:
+
+| Name      | Native | Alias | Description                                   |
+| --------- | ------ | ----- | --------------------------------------------- |
+| `ARRAY` | ✅ |     | An alternative syntax to define one-dimensional arrays. | 
```

Comment:
We can't write ARRAY like this as this may get misinterpreted as a data type. We can skip the table for this data type.